### DISCUSSION TITLE
Can not access student developer pack

### DISCUSSION THREAD

**Discussion by ullask24** [community/community#70717] [GitHub Education]
### Select Topic Area

Question

### Body

Hi, 

despite taking all the steps to qualify for the student developer pack i get the error "Unable to submit your application". Could you please help.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @ullask24 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by scronge** [comment]
Having the same issue. Looks like they no longer review these.
